api/services/StatisticsMassifService.js

```diff
@@ -0,0 +1,231 @@
+const CREATE_MATERIALIZED_VIEW = `
+  CREATE MATERIALIZED VIEW v_massif_info AS
+  SELECT m.id as id_massif,
+  c.id as id_cave,
+  n.name as name_cave,
+  c.depth as depth_cave,
+  c.length as length_cave,
+  c.is_diving as is_diving_cave,
+  COUNT(DISTINCT e.id) as nb_entrances
+  FROM t_massif m 
+  JOIN t_entrance e ON ST_Contains(ST_SetSRID(m.geog_polygon::geometry, 4326), ST_SetSRID(ST_MakePoint(e.longitude, e.latitude), 4326))
+  JOIN t_cave c ON e.id_cave = c.id 
+  JOIN t_name n ON n.id_cave = c.id 
+  WHERE n.is_main = true 
+  AND c.is_deleted = false
+  AND e.is_deleted = false
+  AND m.is_deleted = false
+  GROUP BY(m.id, c.id, n.name, c.depth, c.length, c.is_diving );
+`;
+
+const GET_NB_CAVES = `
+  SELECT COUNT(*) as nb_caves
+  FROM v_massif_info
+  WHERE id_massif = $1
+`;
+
+const GET_NB_NETWORKS = `
+  SELECT COUNT(*) as nb_networks
+  FROM v_massif_info
+  WHERE id_massif = $1
+  AND nb_entrances > 1
```

Comment:
Why do you need `nb_entrances > 1` for this query ?

Reply:
We are trying to retrieve the networks here. We based ourselves on the documentation available on grottocenter-api repo and we found that "Un réseau il s'agit en base de données d'une cave qui est associée à plusieurs entrances."

Reply:
I understand your point !
@urien peux tu confirmer que c'est bien ce qui est attendu pour cette statistique ?

Reply:
Oui c'est cela : les réseaux ce sont toutes les caves qui sont associés à plus d'une entrance